pkg/virt-launcher/virtwrap/device/hostdevice/hostdev.go

```diff
@@ -95,6 +99,27 @@ func createPCIHostDevice(hostDeviceData HostDeviceMetaData, hostPCIAddress strin
 	return domainHostDevice, nil
 }
 
+func createMDEVHostDeviceWithDisplay(hostDeviceData HostDeviceMetaData, mdevUUID string) (*api.HostDevice, error) {
```

Comment:
I just realize now that devices use the converter context to pass int already converted devices. Sounds weird. The idea of the converter was to do all the mapping (to e.g. keep alias generation, ... under a common structure) with the converter context only providing additional runtime parameters which would e.g. requires lookup on the system.

In my opinion the conversion should happen also for devices inside the converter tree. Outside of the scope of this PR.

Reply:
Yea, I agree, this needs to be addressed. I'll rearrange this in another PR.